charts/atlas-advanced/Chart.yaml

```diff
@@ -4,15 +4,15 @@ description: This chart showcases Atlas Advanced (dedicated) deployment
 
 type: application
 
-version: 1.0.0
+version: 2.4.1
 
 keywords:
   - mongodb
   - database
   - nosql
 home: https://github.com/mongodb/mongodb-atlas-kubernetes
 icon: https://webimages.mongodb.com/_com_assets/cms/kuyjf3vea2hg34taa-horizontal_default_slate_blue.svg
-appVersion: "1.0.0"
+appVersion: "2.4.1"
```

Comment:
i think my question still applies: how do we make sure that this does not introduce skew when we move forward with new AKO versions?

Reply:
In general, the helm chart version does not need to match the version of the image.

These examples are even less tied to the Operator releases, they should just keep working after most releases, unless some breaking changes are released.

I see two ways about this:
1) Every Operator and Helm release requires a release in these example charts.
2) These example charts are exercised in a test in the CI and will only get updated as needed, when actual changes makes them somewhat obsolete.

If we do not automate on auto testing these, we will be forced to option 1, at least until we make such investment.

Reply:
According to https://helm.sh/docs/topics/charts/#the-appversion-field:

> Note that the `appVersion` field is not related to the version field. It is a way of specifying the version of the application. For example, the drupal chart may have an appVersion: "8.2.1", indicating that the version of Drupal included in the chart (by default) is 8.2.1. This field is informational, and has no impact on chart version calculations.

With respect to the `version` field: https://helm.sh/docs/topics/charts/#charts-and-versioning

> Every chart must have a version number. A version must follow the [SemVer 2](https://semver.org/spec/v2.0.0.html) standard. Unlike Helm Classic, Helm v2 and later uses version numbers as release markers. Packages in repositories are identified by name plus version.

My conclusions from the above are the following:

1. Given `appVersion` is purely informational I vote for making it clear (as a comment) that this is the version of AKO this helm was created with.

2. Given `version` is a semver version of **this chart** specifically I vote to set it to "1.0" since this is the very first version of this chart

@josvazg @igor-karpukhin thoughts?

Reply:
Sounds reasonable to me.
Also I would avoid having to re-release this with all released of AKO. Releasing only when new versions might force it due to breaking changes or features than need to be showcased.